src/pages/tasks/[id].tsx

```diff
@@ -0,0 +1,16 @@
+import { useRouter } from 'next/router';
+import TaskDetails from '@/components/taskDetails';
+import { TASKS_URL } from '@/components/constants/url';
+
+const TaskDetailsPage = () => {
+    const router = useRouter();
+    const id = router.query?.id as string;
+    const TASK_DETAILS_URL = `${TASKS_URL}/${id}/details`;
+
+    if (!id) {
+        return null;
+    }
```

Comment:
Shouldn't we show an error message to the user, instead of a blank screen?

Reply:
Handled this case with an error message ✅